cmd.go

```diff
@@ -0,0 +1,79 @@
+package gps
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"time"
+)
+
+// monitoredCmd wraps a cmd and will keep monitoring the process until it
+// finishes or a certain amount of time has passed and the command showed
+// no signs of activity.
+type monitoredCmd struct {
+	cmd     *exec.Cmd
+	timeout time.Duration
+	buf     *activityBuffer
+}
+
+func newMonitoredCmd(cmd *exec.Cmd, timeout time.Duration) *monitoredCmd {
+	buf := newActivityBuffer()
+	cmd.Stderr = buf
```

Comment:
Let's track Stderr and Stdout separately - seems like having two *activityBuffers should be fine. Then we'd just need to update hasTimedOut() to check both of them.